### ISSUE TITLE
live server not opening chrome

### ISSUE THREAD

**Issue by roshansap** [ritwickdey/vscode-live-server#816]
I look some settings file but i couldnnt figure it out.. live server works in visual studio code but doesnt work in browser

**Comment by OldIngvar** [comment]
Hello!  I have a similar problem - the browser after starting Live Server does not open the file!  Did you find a solution?

**Comment by OldIngvar** [comment]
> I look some settings file but i couldnnt figure it out.. live server works in visual studio code but doesnt work in browser
> 
> 


Yes, I have the same thing!  I worked with Chrome when there was a problem with Live Server.  I changed all the settings in Live Server to Edge, installed Edge as the default browser on the computer, but Live Server continued to launch Chrome, which did not open the file.  I deleted Chrome from the computer, but Live Server still did not open Edge, and wrote that it could not find Chrome ... The problem has not been resolved ...

**Comment by thelemondropkid** [comment]
@OldIngvar, @roshansap -  Live Server will always use your system's default browser. 
If you want to use a different browser, you have to tell it where it's located on your system. 
To do so on Windows for example, do the following:

In VS Code:
1. File > Preferences > Settings
2. Now use this keyboard shortcut: `Ctrl + shift + p`
3. Select `Preferences: Open Settings (JSON)`
3. Do a search for the following property: `"liveServer.settings.AdvanceCustomBrowserCmdLine"`
4. If you have it, change the value of the property to `"C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe"`
5. If you don't find it, create the property and value in the object like this:
`"liveServer.settings.AdvanceCustomBrowserCmdLine": "C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe"`
